runtime/include/chpl-gen-includes.h

```diff
@@ -35,9 +35,9 @@
 // one argument.
 //
 static inline
-void chpl_ftable_call(chpl_fn_int_t fid, void* arg)
+void chpl_ftable_call(chpl_fn_int_t fid, chpl_comm_on_bundle_t* bundle)
```

Comment:
Interesting!  I would have bet we had calls to chpl_ftable_call() in both the comm and tasking layers, but indeed not; all of them are done from the comm layer.  I can imagine a future in which locale model module code calls this directly for local tasks, but that may never come to pass, so this change is fine as it stands.  I.e., this is just a comment, not a request for a change in the PR.